src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -809,6 +809,12 @@ private void onClosed(Event event) {
         if (tableModel != null) {
             tableModel.unbind();
         }
+
+        // This check is needed, because the WelcomeTab is a tab, but not a database.
+        if (stateManager.getOpenDatabases().isEmpty() && stateManager.activeDatabaseProperty().isPresent().get()) {
+            stateManager.activeDatabaseProperty().setValue(Optional.empty());
+        }
```

Comment:
I don't get why this check is in LibraryTab::onClose . This seems very very odd to me. Shouldn't this go to the "poor-mans" binding in JabRefFrame?